src/include/planner/join_order_enumerator.h

```diff
@@ -101,6 +101,9 @@ class JoinOrderEnumerator {
         std::shared_ptr<NodeExpression> nbrNode, std::shared_ptr<RelExpression> recursiveRel,
         ExtendDirection direction, LogicalPlan& plan);
 
+    void appendLabelFilter(std::shared_ptr<Expression> nodeID,
```

Comment:
Let's be more specific that labels are node labels.
`appendLabelFilter -> appendNodeLabelFilter`